tests/functional/watcherdecisionengine_controller_test.go

```diff
@@ -145,14 +181,41 @@ var _ = Describe("WatcherDecisionEngine controller", func() {
 				corev1.ConditionTrue,
 			)
 		})
-		It("should have ReadyCondition ready", func() {
```

Comment:
not sure if I'm missing it because of the diff visualization, but it looks to me like the check for ReadyCondition is lost now, and we should still check for it

Reply:
no you're right i removed it on last update when working on the tests but i will re-add in the next push